courses-app/src/app/courses-list/courses-list-item/courses-list-item.component.ts

```diff
@@ -30,11 +30,12 @@ export class CoursesListItemComponent implements OnInit {
     } else {
       this.objectStyle['low-rated'] = true;
     }
-
   }
 
   deleteCourse() {
-    this.logId.emit(this.courseData.id);
+    if (confirm('Are you shure?')) {
```

Comment:
Better to use variable for confirm result - easier to understand.
According to task message should be 'Do you really want to delete this course?'.